### ISSUE TITLE
Curious why 4.76 google provider or above is needed for master release.

### ISSUE THREAD

**Issue by shermanericts** [terraform-google-modules/terraform-google-kubernetes-engine#1727]
### TL;DR

I looked at this [PR](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1722/files) and I noticed that the google and google-beta providers are now 4.76.0 and above.

I've had to pin the provider to 4.65 because of this [issue](https://github.com/hashicorp/terraform-provider-google/issues/14679).

I know this is possibly out of scope.  Is there any way to use >= 4.65.0 for the version constraint?  If not, all good - I can see if I can work around the other ticket.  If that's the case you can just close this ticket.

Thank you in advance!

### Expected behavior

See above

### Observed behavior

See above

### Terraform Configuration

```hcl
Terraform Version 1.3.9
google and google-beta 4.65
```

### Terraform Version

```sh
1.3.9
```

**Comment by apeabody** [comment]
Hi @shermanericts - It looks like `>=4.76.0` was required for the addition of `gcs_fuse_csi_driver_config`: https://github.com/hashicorp/terraform-provider-google/releases/tag/v4.76.0
